kartothek/io/dask/dataframe.py

```diff
@@ -301,3 +304,66 @@ def update_dataset_from_ddf(
         metadata=metadata,
         metadata_merger=metadata_merger,
     )
+
+
+def collect_dataset_statistics(
+    store, dataset_uuid, table_name, predicates=None, frac=0.3, factory=None,
+):
+    """
+    Retrieve some statistics, maybe even plot them, generate a report...?
```

Comment:
Just return the data would be enough. The user can do with that whatever they want (or the query planner).

Reply:
- [ ] Adjust docstring: only df